### ISSUE TITLE
Question about iOS

### ISSUE THREAD

**Issue by Slavaleriya** [nanoscopic/stf_ios_support#104]
Hello, please tell me a question about the number of simultaneously connected devices to your project. Is it really possible to connect only 1 iPhone to 1 MacBook, or is it not?

**Comment by nanoscopic** [comment]
As the big stickied issue notes, no one is currently maintaining this project.

I am intending to add STF support to ios_remote_provider in the future as the path forward for STF.

I'll certainly accept commits / PRs to this project by anyone who wishes to contribute, but I see zero reason to exert any of my own development time into this project. It's dead in the water.

This project does theoretically support two devices on a single MacOS machine. I've tested it previously ( when it was easy to get it running ) and it worked.

There are a bunch of reasons why this project doesn't support more than two devices per machine:

1. The way video is fetched/streamed from the device is by getting a H264 stream from device and decoding it on the MacOS machine. Most machines can only do this in a hardware accelerated fashion for a single stream at once. A second stream must decode using software. Decoding each stream in software takes around 70% of an older mac mini core. So, perhaps 3 or even 4 simultaneous devices could work, but it will max out the CPU.

2. There are lots of bugs in stf_ios_support that I only fixed by completely re-writing everything ( and hence ending up with ios_remote_provider ). Those bugs could be fixed here, but this project was created while working for TMobile. They, therefore, ultimately own the Copyright for this project. I want to focus on a project which I own, not on one owned by someone else. Hence the full rewrite from scratch. Essentially, I have no motivation or reason to support stf_ios_support any more. It doesn't benefit me.

3. The way video is done in STF is bad/outdated architecturally. STF doesn't support proper encryption or security. It's unsafe to use it for sensitive video content without using a VPN. Even if you one spend the effort to fix the problems with this project, unless you also change major portions of STF it is wasted effort. I consider STF to be legacy software. One should only use it for the purpose where there is no better alternative ( for Android devices ). For iOS devices, ControlFloor is a far better solution, despite CF still being, essentially, in alpha.

--

In closing, if you are still trying to make STF work with iOS devices, you are beating a dead animal. You'll get diminishing returns the more effort you spend down that path.
